apps/mobile-wallet/src/components/WalletSwitchButton.tsx

```diff
@@ -64,6 +64,7 @@ const WalletSwitchButton = ({ style }: WalletSwitchButtonProps) => {
       setNbOfTaps(0)
     }
 
+    // TODO: Shouldn't this be 69?
```

Comment:
😨 you're right. I wanted to start counting from -1 to make it less obvious, but apparently I considered this as non vital and made a mistake 🙃